mlir/test/Dialect/Tosa/profile_bi_unsupported.mlir

```diff
@@ -0,0 +1,26 @@
+//--------------------------------------------------------------------------------------------------
```

Comment:
nit: filename should be updated to pro_int

Reply:
Done.